proto/interchain_security/ccv/provider/v1/query.proto

```diff
@@ -426,3 +434,16 @@ message QueryConsumerIdFromClientIdResponse {
   // the consumer id of the chain associated with this client id
   string consumer_id = 1;
 }
+
+message QueryConsumerChainRequest {
+  string consumer_id = 1;
+}
+
+message QueryConsumerChainResponse {
+  string chain_id = 1;
+  string owner_address = 2;
+  uint32 phase = 3;
```

Comment:
Would be useful the phase to be a string for users to not look for the meaning of the numbers.

Reply:
Done.